VirtoCommerce.InventoryModule.Web/Controllers/Api/InventoryModuleController.cs

```diff
@@ -56,22 +52,35 @@ public IHttpActionResult GetFulfillmentCenter(string id)
             return Ok(retVal);
         }
 
+        /// <summary>
+        /// Get fulfillment centers by ids
+        /// </summary>
+        /// <param name="ids">fulfillment center ids</param>
+        [HttpPost]
+        [ResponseType(typeof(FulfillmentCenter[]))]
+        [Route("fulfillmentcenters/plenty")]
+        public IHttpActionResult GetFulfillmentCenters([FromBody] string[] ids)
+        {
+            var retVal = _fulfillmentCenterService.GetByIds(ids);
+            return Ok(retVal);
+        }
+
         /// <summary>
         ///  Save fulfillment center 
         /// </summary>
-        /// <param name="center">fulfillment center</param>
+        /// <param name="centers">fulfillment center</param>
         [HttpPut]
-        [ResponseType(typeof(FulfillmentCenter))]
+        [ResponseType(typeof(FulfillmentCenter[]))]
         [Route("fulfillmentcenters")]
         [CheckPermission(Permission = InventoryPredefinedPermissions.FulfillmentEdit)]
-        public IHttpActionResult SaveFulfillmentCenter(FulfillmentCenter center)
+        public IHttpActionResult SaveFulfillmentCenter(FulfillmentCenter[] centers)
```

Comment:
It is 100% breaking changes. The new bulk update for fulfillment centers must be done as a new endpoint. Old endpoint PUT api/inventory/fulfillment centers must be unchanged and has the same signature.

Reply:
It was done according to the previous comment:
Could we leave only one API method which received an array of fulfillment centers? (requires to manager UI changes)

Reply:
After discussion, we decided to leave the original method too for backward compatibility.

Reply:
fixed